api/chunks/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from pgvector.django import VectorField
+
+class Chunk(models.Model):
+    text = models.CharField(max_length=100)
+    embedding = VectorField(dimensions=10)
+    chunk_idx = models.IntegerField()
+    start_char = models.IntegerField()
+    end_char = models.IntegerField()
```

Comment:
**issue:** please add typing, it looks something like this
```python
text = models.CharField[str, str](max_length=100)
```
note that you have to add type twice, because it's input/output